ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/UPLINK_MLAG_STRUCTURED_CONFIG_L2LEAFS.yml

```diff
@@ -30,7 +30,9 @@ l2leaf:
           id: 1
           mgmt_ip: 192.168.201.201/24
           uplink_switch_interfaces: [ Ethernet5, Ethernet5 ]
+          mgmt_gateway: 1.1.1.3
```

Comment:
I would prefer if we did not put this in the UPLINK_MLAG _.. test as well. That makes it difficult to know what we are testing where as the test is hidden under a file with a completely unrelated name

Create a new host inside a node_group to make it clear what we are testing.

Reply:
Added new hosts to a new group for testing this feature